buildenv/docker/mkdocker.sh

```diff
@@ -57,9 +59,13 @@ fi
 # Global configuration variables.
 action=print
 arch=
+bootjdks=
```

Comment:
Suggest better names for `bootjdks` and `jdks_to_build` are `bootjdk_versions` and `jdk_versions`.
The default `jdk_versions=all` should appear here.
`extensionrepos` is no longer needed.